packages/opds-web-client/src/components/__tests__/Root-test.tsx

```diff
@@ -755,4 +756,71 @@ describe("Root", () => {
       expect(push.args[0][0]).to.equal(context.pathFor(collectionUrl, null));
     });
   });
-});
\ No newline at end of file
+
+  describe("provided a EntryPointsTabs", () => {
+    class Tabs extends React.Component<EntryPointsTabsProps, any> {
```

Comment:
This is a nitpick but that `any` can be `void` here.

Reply:
I'm still learning Typescript, so don't worry about being nitpicky.

Reply:
That second type is for the state, and this component has no state.